js/main.js

```diff
@@ -1,123 +1,73 @@
+        .done(function(data, status) {
+            console.log('DONE: Status is ', status)
+            renderReport(data);
+        })
+        .fail(function(xhr, status, err) {
+            console.error('fail', status, err);
+        });
+
+    $.ajax({method: 'GET', url: 'https://data.austintexas.gov/api/views/8qty-vat2.json'})
+        .done(function(data, status) {
+            console.log('DONE: Status is ', status)
+            renderReportMetadata(data);
+        })
+        .fail(function(xhr, status, err) {
+            console.error('fail', status, err);
+        });
 }
 
-function findLastUpdate(data){
-    var date = new Date(data.rowsUpdatedAt * 1000);
-    $(".updated-date").text(date.toDateString());
-}
-
-function fillCards(data){
-
-    for (var i = 0; i < data.length; i++) {
-      var $container       = $(".container"),
-          cardDeptName     = ".card-" + i + " .dept-name",
-          liasonScore      = ".card-" + i + " .liaison-score",
-          inventoryScore   = ".card-" + i + " .inventory-score",
-          plansScore       = ".card-" + i + " .plans-score",
-          publicationScore = ".card-" + i + " .publication-score";
-
-      $container.append(htmlBlock);
-      $container.children().last().addClass("card-"+i);
-
-
-      if ( data[i].liaison === "completed" ){
-          $(liasonScore).addClass("green");
-      }
-
-      if ( data[i].inventory === "submitted" ){
-          $(inventoryScore).addClass("green");
-      } else if ( data[i].inventory === "in progress" ){
-          $(inventoryScore).addClass("yellow");
-      } else if ( data[i].inventory === "not started" ){
-          $(inventoryScore).addClass("red");
-      }
-
-      if ( data[i].plans === "submitted" ){
-          $(plansScore).addClass("green");
-      } else if ( data[i].plans === "not yet" ){
-          $(plansScore).addClass("red");
-      }
-
-      if ( data[i].publication === "completed" ){
-          $(publicationScore).addClass("green");
-      } else if ( data[i].publication === "in progress" ){
-          $(publicationScore).addClass("yellow");
-      } else if ( data[i].publication === "unknown" ){
-          $(publicationScore).addClass("red");
-      }
-
-
-
-      $(cardDeptName).text(data[i].dept)
-      $(liasonScore + " .score-text").text(data[i].liaison)
-      $(inventoryScore + " .score-text").text(data[i].inventory)
-      $(plansScore + " .score-text").text(data[i].plans)
-      $(publicationScore + " .score-text").text(data[i].publication)
-
-      var imgFilename = data[i].dept.toLowerCase().replace(/\s+/g, "-");
-      var imgPath = "img/" + imgFilename + ".svg";
-
-      $(".card-" + i + " .dept-icon").attr("src", imgPath)
 
+function renderReportMetadata(data) {
+    var source = $('#metadata-template').html();
+    var template = Handlebars.compile(source);
+    var context = {
+        updated: moment.unix(data.rowsUpdatedAt).fromNow()
```

Comment:
Good call on moment.js. Time since update is a nice touch instead of just the date.